### ISSUE TITLE
Condenser and hopper

### ISSUE THREAD

**Issue by mtdudek** [Bartz24/SkyResources#129]
Hi,

I'm playing MSB3 and I have annoying issue. Every time I comeback  from nether, hopper starts pulling alchemical ore dust from condenser. Only way to fix it is to break and replace condenser.

Thanks for response.

**Comment by Bartz24** [comment]
I just tested this coming back from the nether. It still continued to work correctly

**Comment by mtdudek** [comment]
I don't know, maybe I will send you my save.
Edit:
here is youtube video https://youtu.be/4mxjX3QJ6Ms

**Comment by FusionLord** [comment]
For starters the condenser auto puts into the container below it.
Secondly what version of MSB3 and of SR2?

**Comment by mtdudek** [comment]
MSB3->3.1.5.3
SR2->2.2.3
I know that condenser is pushing crafted items below itself, but it shouldn't allow hoppers to take alchemical dust from its slot.

**Comment by FusionLord** [comment]
I was only mentioning it to let you know y out don't need the hopper at all

**Comment by 630R63** [comment]
Just happened to me right now and I was looking to see if an issue is already opened on this topic.
It happened without going to the nether... I just logged in on the world and tried to create some iron ingots from the Iron Alchemical Dust using a stone casing with a stone condenser and having a Wooden Hopper underneath from the Tinker's Skyblock mod. I had to rebuild the condenser in order to fix it, so it doesn't seem to be a hopper's issue.

Yesterday when I was playing the pack it wasn't doing this weird bug. 
Just tried to reproduce it on a creative world and I was unsuccessful as well. 

MSB3 v.3.3.1
SR2 v.2.3.0
